prototypes/technology/vehicle-equipment.lua

```diff
@@ -18,77 +18,74 @@ local inputs = {
-    ["vehicle-laser-defense-equipment-3"] = { group = "equipment", icon_name = "laser-defense", tier = 2 },
-    ["vehicle-laser-defense-equipment-4"] = { group = "equipment", icon_name = "laser-defense", tier = 3 },
-    ["vehicle-laser-defense-equipment-5"] = { group = "equipment", icon_name = "laser-defense", tier = 4 },
-    ["vehicle-laser-defense-equipment-6"] = { group = "equipment", icon_name = "laser-defense", tier = 5 },
+    ["bob-vehicle-laser-defense-equipment-1"] = { group = "equipment", icon_name = "laser-defense", tier = 0 },
+    ["bob-vehicle-laser-defense-equipment-2"] = { group = "equipment", icon_name = "laser-defense", tier = 1 },
+    ["bob-vehicle-laser-defense-equipment-3"] = { group = "equipment", icon_name = "laser-defense", tier = 2 },
+    ["bob-vehicle-laser-defense-equipment-4"] = { group = "equipment", icon_name = "laser-defense", tier = 3 },
+    ["bob-vehicle-laser-defense-equipment-5"] = { group = "equipment", icon_name = "laser-defense", tier = 4 },
+    ["bob-vehicle-laser-defense-equipment-6"] = { group = "equipment", icon_name = "laser-defense", tier = 5 },
 
     -- Fusion reactors
-    ["vehicle-fusion-reactor-equipment-1"] = { group = "equipment", icon_name = "fusion-reactor", tier = 0 },
-    ["vehicle-fusion-reactor-equipment-2"] = { group = "equipment", icon_name = "fusion-reactor", tier = 1 },
-    ["vehicle-fusion-reactor-equipment-3"] = { group = "equipment", icon_name = "fusion-reactor", tier = 2 },
-    ["vehicle-fusion-reactor-equipment-4"] = { group = "equipment", icon_name = "fusion-reactor", tier = 3 },
-    ["vehicle-fusion-reactor-equipment-5"] = { group = "equipment", icon_name = "fusion-reactor", tier = 4 },
-    ["vehicle-fusion-reactor-equipment-6"] = { group = "equipment", icon_name = "fusion-reactor", tier = 5 },
+    ["bob-vehicle-fission-reactor-equipment-1"] = { group = "equipment", icon_name = "fission-reactor", tier = 0 },
+    ["bob-vehicle-fission-reactor-equipment-2"] = { group = "equipment", icon_name = "fission-reactor", tier = 1 },
+    ["bob-vehicle-fission-reactor-equipment-3"] = { group = "equipment", icon_name = "fission-reactor", tier = 2 },
+    ["bob-vehicle-fission-reactor-equipment-4"] = { group = "equipment", icon_name = "fission-reactor", tier = 3 },
+    ["bob-vehicle-fission-reactor-equipment-5"] = { group = "equipment", icon_name = "fission-reactor", tier = 4 },
+    ["bob-vehicle-fission-reactor-equipment-6"] = { group = "equipment", icon_name = "fission-reactor", tier = 5 },
 
     -- Mobility
-    ["vehicle-motor-equipment"] = { flat_icon = true },
-    ["vehicle-engine-equipment"] = { flat_icon = true },
-    ["vehicle-belt-immunity-equipment"] = { flat_icon = true },
+    ["bob-vehicle-motor-equipment"] = { flat_icon = true },
+    ["bob-vehicle-engine-equipment"] = { flat_icon = true },
+    ["bob-vehicle-belt-immunity-equipment"] = { flat_icon = true },
```

Comment:
The belt immunity equipment is not being handled correctly, the sprite when placed on the ground is the old Bob-managed sprite with background.